libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -669,7 +669,7 @@ namespace impl {
        * and can be degraded to the normal ::from_variant(...) processing
        */
       template<typename M, typename Resolver, not_require_abi_t<M> = 1>
-      static void extract( const variant& v, M& o, Resolver, abi_traverse_context& ctx )
+      static void extract( const fc::variant& v, M& o, Resolver, abi_traverse_context& ctx )
```

Comment:
Is fc::variant kept here due to the api consistency? using std::variant will cause much changing on the user side?

Reply:
`fc::variant` wasn't a consideration for this effort.  Bart wanted only `fc:::static_varaint` taken out.

Reply:
cool, thanks for explanation